scripts/pr-convergence.py

```
@@ -410,6 +414,92 @@ def graphql_open_pulls_by_head_ref(self, head_ref_name: str) -> list[dict[str, A
+
+    try:
+        sibling_prs = client.graphql_open_pulls_by_head_ref(base_ref)
+    except GhReadError as exc:
+        errors.append({"source": "sibling_prs", "message": str(exc)})
+        return result
+
+    # Filter: only PRs targeting the same base that are NOT this PR and NOT
+    # a stack parent/child (different head ref targeting the same base).
+    siblings = []
+    for candidate in sibling_prs:
+        candidate = as_dict(candidate)
```

Ruff (0.16.0)

[warning] `for` loop variable `candidate` overwritten by assignment target

(PLW2901)